### PR TITLE
menon2021re v1

### PR DESCRIPTION
**Pull request by kevins99**
Hello, please find an initial version of the Camera ready paper. No errors on our end while formatting, just a few warnings. 
Thank you!

### CODE REVIEW

File: menon2021re/journal/metadata.yaml [hunk 1]
```diff
@@ -55,8 +40,8 @@ keywords: rescience c, rescience x
 # Software Heritage.
 #   see https://guides.github.com/activities/citable-code/
 code:
-  - url: https://github.com/rescience-c/template
-  - doi: 
+  - url: https:/github.com/alchemi5t/MNADrc
+  - doi: 10.5281/zenodo.4681515
   - swh:
```
Comment (koustuvsinha) [inline menon2021re/journal/metadata.yaml:45]: Please add the Software Heritage Foundation URI for your code (instructions in README)

File: menon2021re/journal/metadata.yaml [hunk 2]
```diff
@@ -66,22 +51,33 @@ data:
...
-           and since September 2015, things have been going steadily. We're
-           still alive, independent and without a budget. In the meantime, we
-           have published around 20 articles (mostly in computational
-           neuroscience & computational ecology) and the initial has grown
-           from around 10 to roughly 100 members (editors and reviewers), we
-           have advertised ReScience at several conferences worldwide, gave
-           some interviews and we published an article introducing ReScience in
-           PeerJ. Based on our experience at managing the journal during these
-           three years, we think the time is ripe for proposing some changes."
+abstract: "Scope of Reproducibility
```
Comment (koustuvsinha) [inline menon2021re/journal/metadata.yaml:60]: You can probably remove the line breaks from the abstract here.

**Comment by kevins99** [comment]
HI!
Have made the following changes:
1) Added SWH in metadata.yaml
2) Removed line breaks from abstract in metadata.yaml
Thank you!

**Comment by koustuvsinha** [comment]
Compilation works on my end. However, I noticed you have removed the symlinks and added the folders from template directly. Please revert that, as the symlinks help to centralize any update to the main template in one place. From your journal folder, clean up the PR using the following:

```
rm -rf Makefile article.py header.tex rescience.cls roboto source-code-pro source-sans-pro source-serif-pro yaml-to-bibtex.py yaml-to-latex.py yaml-to-markdown.py metadata.tex
```

And then add the symlinks:

```
    ln -s ../../template/Makefile .
    ln -s ../../template/article.py .
    ln -s ../../template/header.tex .
    ln -s ../../template/rescience.cls .
    ln -s ../../template/roboto .
    ln -s ../../template/source-code-pro .
    ln -s ../../template/source-sans-pro .
    ln -s ../../template/source-serif-pro .
    ln -s ../../template/yaml-to-bibtex.py .
    ln -s ../../template/yaml-to-latex.py .
    ln -s ../../template/yaml-to-markdown.py .
```

**Comment by kevins99** [comment]
We have incorporated the suggested changes regarding symlinks. 
However, post adding the symlinks and deleting the required folders, we onoticed that our names were bot being displayed. 
We fixed this by setting the abstract in metadata.yaml to " "(Empty string). We hope that will not be an issue and apologise for the delay.

**Comment by koustuvsinha** [comment]
Looks good to me. I also reviewed the pdf content, looks good.

**Pull request merged**